### PR TITLE
Create schema.py

### PR DESCRIPTION
**Pull request by GermanParra**
## Description

https://www.loom.com/share/d07a727e586f49e0b45f706b32ee21ec

Created schema.py file with pydantic schemas.

Note: The schemas include all the collected information from mentees and mentors but can be easily changed to omit unnecessary fields, after been implemented in the API endpoints.

## Type of change

- [x] New feature

## Checklist:

- [x] My code follows PEP8 style guide
- [x] I have removed unnecessary print statements from my code
- [x] I have made corresponding changes to the documentation if necessary
- [x] My changes generate no errors
- [x] No commented-out code
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by BrokenShell — CHANGES_REQUESTED** [review]
Great job over all. However, I think we need to make some changes based on our last conversation with Jason. Let's run through updates in Office Hours.

**Review by mikayla-kosmala — APPROVED** [review]
Glad to see that all the changes from the conversation with Jason we implemented! I do worry about the ones we took out, but that can be implemented later :)

File: app/schema.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+from datetime import datetime
+from pydantic import BaseModel, constr
+
+
+class Variants:
+    exp_levels = Literal[
+        "Beginner", "Intermediate", "Advanced"
+    ]
+
+    teck_stacks = Literal[
```
Comment (BrokenShell) [inline app/schema.py:11]: `tech_stack` not `teck_stack`

Comment (GermanParra) [reply app/schema.py]: I just fixed, thank you.

**Review by jinjahninjah — CHANGES_REQUESTED** [review]
Don't forget to refactor any code that uses the tech_stack literal 👍

**Comment by GermanParra** [comment]
> Don't forget to refactor any code that uses the tech_stack literal 👍
I just did thank you 👍

**Review by jinjahninjah — APPROVED** [review]

**Comment by derekjpeters** [comment]
[Test Card 1](https://trello.com/c/ppyR7Rpj/7-test-card-1)

**Pull request merged**